packages/cloud-cognitive/src/components/AddSelect/AddSelect.js

```diff
@@ -303,4 +323,10 @@ AddSelect.propTypes = {
   title: PropTypes.string,
 };
 
+AddSelect.defaultProps = {
```

Comment:
Should we stick to using a `defaults` object? How come you added back `defaultProps`?

Reply:
i didn't realize that was the new standard. i can update that going forward